dfinity/nns-dapp
# CHANGELOG for release on 2022-05-19

# Motivation
There has been a release of the nns-dapp: https://dashboard.internetcomputer.org/proposal/60694

## Details

**Title:** Upgrade frontend NNS Dapp canister to commit `4a63f25d540ccac94968dc7d29cc70b60ecd5a9b`
**Hash and wasm source:** Wasm sha256 hash: `c6b16dc9fa2f330899c20f44d0477d38498688bd0c56b56273b753a64cf74331` (`https://github.com/dfinity/nns-dapp/actions/runs/2351043904`)

SHA256:
- CI: c6b16dc9fa2f330899c20f44d0477d38498688bd0c56b56273b753a64cf74331  nns-dapp.wasm
- max: c6b16dc9fa2f330899c20f44d0477d38498688bd0c56b56273b753a64cf74331

## Change Log:

- Update mechanics for creating canisters
- Prepare to release the neuron and accounts tabs in svelte


# Changes
Update the changelog

# Tests
- Verify that canisters can be created
- Verify that canisters can be topped up
